drivers/adc/adc_context.h

```diff
@@ -205,16 +210,16 @@ static inline void adc_context_start_read(struct adc_context *ctx,
 static inline void adc_context_on_sampling_done(struct adc_context *ctx,
 						struct device *dev)
 {
-	if (ctx->sequence->options) {
+	if (ctx->sequence.options) {
 		adc_sequence_callback callback =
-			ctx->sequence->options->callback;
+			ctx->options.callback;
```

Comment:
After the change, this line no longer needs to be broken in two.

Reply:
done